vips/vip-160/vip-160.ts

```diff
@@ -0,0 +1,175 @@
+    version: "v2",
+    title: "VIP-160 Add THE and TWT market to DeFi Pool",
+    description: ``,
+    forDescription: "I agree that Venus Protocol should proceed with add market",
+    againstDescription: "I do not think that Venus Protocol should proceed with add market",
+    abstainDescription: "I am indifferent to whether Venus Protocol proceeds with add market",
+  };
+
+  return makeProposal(
+    [
+      // Tranfer From Treasury to community wallet
+      {
+        target: TREASURY,
+        signature: "withdrawTreasuryBEP20(address,uint256,address)",
+        params: [USDT, parseUnits("6000", 18), COMMUNITY_WALLET],
+      },
+      // ================ THE Market ========================
+      {
+        target: BINANCE_ORACLE,
+        signature: "setMaxStalePeriod(string,uint256)",
+        params: ["THE", MAX_STALE_PERIOD],
+      },
+      {
+        target: RESILIENT_ORACLE,
+        signature: "setTokenConfig((address,address[3],bool[3]))",
+        params: [
+          [
+            THE,
+            [
+              BINANCE_ORACLE,
+              "0x0000000000000000000000000000000000000000",
+              "0x0000000000000000000000000000000000000000",
+            ],
+            [true, false, false],
+          ],
+        ],
+      },
+      {
+        target: TREASURY,
+        signature: "withdrawTreasuryBEP20(address,uint256,address)",
+        params: [THE, parseUnits("58823.5", 18), NORMAL_TIMELOCK],
+      },
+      {
+        target: THE,
+        signature: "approve(address,uint256)",
+        params: [POOL_REGISTRY, 0],
+      },
+      {
+        target: THE,
+        signature: "approve(address,uint256)",
+        params: [POOL_REGISTRY, parseUnits("58823.5", 18)],
+      },
+      {
+        target: POOL_REGISTRY,
+        signature: "addMarket((address,uint256,uint256,uint256,address,uint256,uint256))",
+        params: [
+          [
+            VTHE_DeFi,
+            parseUnits("0.2", 18),
+            parseUnits("0.3", 18),
+            parseUnits("58823.5", 18),
+            VTOKEN_RECEIVER_THE,
+            parseUnits("2600000", 18),
+            parseUnits("1400000", 18),
+          ],
+        ],
+      },
+      // ================= TWT Market ======================
+      {
+        target: BINANCE_ORACLE,
+        signature: "setMaxStalePeriod(string,uint256)",
+        params: ["TWT", MAX_STALE_PERIOD],
+      },
+      {
+        target: RESILIENT_ORACLE,
+        signature: "setTokenConfig((address,address[3],bool[3]))",
+        params: [
+          [
+            TWT,
+            [
+              BINANCE_ORACLE,
+              "0x0000000000000000000000000000000000000000",
+              "0x0000000000000000000000000000000000000000",
+            ],
+            [true, false, false],
+          ],
+        ],
+      },
+      {
+        target: TREASURY,
+        signature: "withdrawTreasuryBEP20(address,uint256,address)",
+        params: [TWT, parseUnits("58823.5", 18), NORMAL_TIMELOCK],
```

Comment:
We already have the bootstrap liquidity of TWT in the treasury: 10,000 TWT

Reply:
Changed in #62